pkg/registrar/registrar.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Actually, the registrar is only used to make sure a name is unique, so that we won't create the same container/sandbox multiple times. I'll simplify this to a simple name to id mapping, name must be unique, and all operation should use name.